File: app/src/main/java/ltd/kaizo/mynews/controller/ui/news/DetailActivity.java
package ltd.kaizo.mynews.controller.ui.news;

import android.content.Intent;
import android.os.Bundle;
import androidx.appcompat.app.ActionBar;
import androidx.appcompat.app.AppCompatActivity;
import androidx.appcompat.widget.Toolbar;
import android.webkit.WebView;

import butterknife.BindView;
import butterknife.ButterKnife;
import ltd.kaizo.mynews.R;

/**
 * The type Detail activity.
 */
public class DetailActivity extends AppCompatActivity {
    /**
     * The Toolbar.
     */
    @BindView(R.id.activity_detail_toolbar)
    Toolbar toolbar;
    /**
     * The Web view.
     */
    @BindView(R.id.activity_detail_webview)
    WebView webView;
    /**
     * The Article url.
     */
    private String articleUrl = "";

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_detail);
        //bind view
        ButterKnife.bind(this);
        this.configureToolbar();
        //get the url from the intent
        Intent intent = getIntent();
        articleUrl = intent.getStringExtra("articleUrl");
        //load the webView with the url
        webView.loadUrl(articleUrl);
    }

    /**
     * Configure toolbar.
     */
    private void configureToolbar() {
        setSupportActionBar(toolbar);
        ActionBar ab = getSupportActionBar();
        if (ab != null)
            ab.setDisplayHomeAsUpEnabled(true);

    }
}

File: app/src/main/java/ltd/kaizo/mynews/controller/ui/news/DetailActivity.kt
package ltd.kaizo.mynews.controller.ui.news

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import kotlinx.android.synthetic.main.activity_detail.*
import ltd.kaizo.mynews.R

class DetailActivity : AppCompatActivity() {
    private var articleUrl = ""
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_detail)
        this.configureToolbar()
        //get the url from the intent
        val intent = intent
        articleUrl = intent.getStringExtra("articleUrl")
        //load the webView with the url
        activity_detail_webview.loadUrl(articleUrl)
    }

    /**
     * Configure toolbar.
     */
    private fun configureToolbar() {
        setSupportActionBar(activity_detail_toolbar)
        val ab = supportActionBar
        ab?.setDisplayHomeAsUpEnabled(true)

    }
}
